.gitignore

```diff
@@ -122,3 +122,8 @@ npm-shrinkwrap.json
 .volumes
 
 cypress
+
+packages/web/app/next.config.mjs
+packages/web/app/environment-*.mjs
+packages/web/app/src/gql/*.ts
+packages/web/app/src/graphql/*.ts
```

Comment:
it's better to put theme here instead of `packages/web/app/src/gql/.gitignore` or `packages/web/app/src/graphql/.gitignore` because ESLint is configured to respect ignore everything from root `.gitignore`